kickstarts/partials/packages/excludes.ks.erb

```diff
@@ -9,15 +9,23 @@
 -ipw2200-firmware
 -ivtv-firmware
 -iwl100-firmware
+-iwl105-firmware
+-iwl135-firmware
 -iwl1000-firmware
+-iwl2000-firmware
+-iwl2030-firmware
+-iwl3160-firmware
 -iwl3945-firmware
 -iwl4965-firmware
 -iwl5000-firmware
 -iwl5150-firmware
 -iwl6000-firmware
 -iwl6000g2a-firmware
+-iwl6000g2b-firmware
 -iwl6050-firmware
-#kernel-firmware
+-iwl7260-firmware
+-iwl7265-firmware
+-kernel-firmware
```

Comment:
@jrafanie if a firmware was needed, the appliance when it starts up would not be able to correctly use a virtual device. If the device was a network card, the appliance would no be able to connect to the network for example. The build would not fail.

But note that these firmwares are for Wifi cards, Fibrechannel adapters, etc. VMs never have these kinds of devices. All network and disk devices for hypervisors that we support are simple (virtio, ide, etc), and they are all supported out of the box without any firmware.